packages/ibm-products-web-components/examples/options-tile/index.html

```diff
@@ -0,0 +1,47 @@
+<!--
+@license
+
+Copyright IBM Corp. 2025
+
+This source code is licensed under the Apache-2.0 license found in the
+LICENSE file in the root directory of this source tree.
+-->
+
+<html>
+  <head>
+    <title>carbon-web-components example</title>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="src/styles.scss" />
+    <style>
+      /* Suppress custom element until styles are loaded */
+      cds-ai-label:not(:defined) {
+        display: none;
+      }
+    </style>
+    <script type="module" src="src/index.js"></script>
+  </head>
+  <body>
+    <c4p-options-tile
+      id="my-tile"
+      size="lg"
+      title="The title"
```

Comment:
Let’s make the options tile example a little bit more contextual. I would be okay with us following the Storybook example (e.g. Auto recovery), but if we want to be consistent with existing Stackblitz examples ... maybe something like:

```suggestion
      title="A title describing all included content"
```